pkg/infrastructure/powervs/clusterapi/powervs.go

```diff
@@ -163,6 +163,19 @@ func (p Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
 	if err != nil {
 		return fmt.Errorf("failed to add ping security group rule: %w", err)
 	}
+
+	if in.InstallConfig.Config.Publish == types.InternalPublishingStrategy &&
+		(len(in.InstallConfig.Config.ImageDigestSources) > 0 || len(in.InstallConfig.Config.DeprecatedImageContentSources) > 0) {
+		vpcID := *powerVSCluster.Status.VPC.ID
+		logrus.Debugf("InfraReady: Ensuring VPE gateway for IAM in VPC %v", vpcID)
```

Comment:
This log is inaccurate, make it clear that multiple gateways are being created (IAM, COS, DNS)

Reply:
Corrected in the latest force-push. PTAL.